### ISSUE TITLE
synonyms and  readouts/ responses  for  PTI signalling

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#18038]
We found the PTI signalling term yesterday, hooray!

so when this is done:
https://github.com/geneontology/go-ontology/issues/18033

please add synonyms to 
GO:0002221 pattern recognition receptor signalling pathway

Pathogen triggered immunity signalling
PAMP triggered immunity signalling
PTI signalling

this definition probably has a problem:
Definition (GO:0002221 GONUTS page)
Any series of molecular signals generated as a consequence of a pattern recognition receptor (PRR) binding to one of its physiological ligands. PRRs bind pathogen-associated molecular pattern (PAMPs), structures conserved among microbial species, or damage-associated molecular pattern (DAMPs), endogenous molecules released from damaged cells. PMID:15199967

That  the processes for PAMPS and DAMPS appear to be different.
We need to find out oif these are the same processes/pathway

DAMP are damage-associate receptors that trigger an inflamatory response so should not be in this term
DAMPs include things like host-produced heat-shock proteins

The PTI trigged immunity pathway has
MAP kinase signalling
Is a innate immune response 
is a local response

Other features of plant PTI (are  any of these true for any other species that use this term?)

- Involved in "FIRST LAYER OF DEFENSE" (need ref)

- Activates RoS (which in plants triggers deposition of lignin and callose to thicken cell walls). Here we would make a concurrent annotation to "positive regulation of callose deposition" or some appropriate term. 
-  expression of PTI-related pathogen resistance (PR) genes

**Comment by ValWood** [comment]
This term might be problematic longer term We probably need different flavors for animals and plants.
Here the term is very non-specific for a process (it is describing any type of  signalling activated by a PRR receptor,  and so is no different from any  "immune response signalling".

It would be good to get a common differentia on this  term. Do they always signal through MAP kinase pathways?

This is a very nice summary of the pathway in plants
https://www.sciencedirect.com/science/article/pii/S1674205215000878#fig1

**Comment by ValWood** [comment]
@hattrill  how does this compare  with PTI in flys.
We also need to get mammalian input too?

**Comment by hattrill** [comment]
So, we would just class this as an innate immune response and might use the type of pathogen it's against e.g.  antifungal innate immune response 

We then co-annotate to indicate which pathway is involved - for flies, this is primarily via the NF-kappaB pathways (PMID:20485470):
Toll signaling pathway
Imd Signaling Pathway

**Comment by hattrill** [comment]
the 'peptidoglycan recognition protein signaling pathway' GO:0061057 could come under GO:0002221 pattern recognition receptor signalling pathway/cell surface pattern recognition receptor signaling pathway GO:0002752
The 'Toll signaling pathway' in flies is also a developmental pathway, so not so easy.

**Comment by pgaudet** [comment]
@hattrill I suppose the immune vs developmental Toll pathways must be different pathways - possibly even triggered by different stimuli ? Should we not distinguish those with 2 separate terms ? 

Pascale

**Comment by ValWood** [comment]
the term 

'peptidoglycan recognition protein signaling pathway' 
seems very general but ti is quite a specific defense response pathway?
Would one of the exact  synonyms  which mentions "immune signaling" make a better primary term name?

Imd signalling pathway | exact
-- | --
immune deficiency pathway | related
Imd signaling pathway | exact
PGRP signaling pathway | related
immune deficiency signaling pathway | related

**Comment by hattrill** [comment]
Yes, it is...... 'Imd signaling pathway' would be the most frequently used in the literature, followed by 'immune deficiency pathway' (practically never 'peptidoglycan recognition protein signaling pathway' - as you say, it is sounds very general, could be mistaken for any pathway with a PGRP-activated receptor) - a name change would be appreciated.
The def refers specifically to Imd:
"A series of molecular signals initiated by binding of peptidoglycan to a receptor on the surface of the target cell and ending with regulation of a downstream cellular process. The main outcome of the Imd signaling is the production of antimicrobial peptides."

**Comment by hattrill** [comment]
> @hattrill I suppose the immune vs developmental Toll pathways must be different pathways - possibly even triggered by different stimuli ? Should we not distinguish those with 2 separate terms ?
> 
> Pascale

Yes, I have had similar thoughts (this is about to have its birthday: https://github.com/geneontology/go-ontology/issues/16672)......but I might have to go and remind myself about them!
